### ISSUE TITLE
Unable to install on Centos7/php7.3

### ISSUE THREAD

**Issue by mickeybyte** [ltb-project/self-service-password#308]
Hi,

I'm trying to install the Self Service Password on my CentOS 7 system, which is already running apache with php7.3 (remi repository).
I installed the requirements for php 7.3 (ldap, mbstring, openssl) but when I try to run the installation via yum it complains it needs php-5.4 modules:

```
Resolving Dependencies
--> Running transaction check
---> Package self-service-password.noarch 0:1.3-1.el7 will be installed
--> Processing Dependency: php-mbstring for package: self-service-password-1.3-1.el7.noarch
--> Processing Dependency: php-ldap for package: self-service-password-1.3-1.el7.noarch
--> Running transaction check
---> Package php-ldap.x86_64 0:5.4.45-17.el7.remi will be installed
--> Processing Dependency: php-common(x86-64) = 5.4.45-17.el7.remi for package: php-ldap-5.4.45-17.el7.remi.x86_64
---> Package php-mbstring.x86_64 0:5.4.45-17.el7.remi will be installed
--> Processing Dependency: php-common(x86-64) = 5.4.45-17.el7.remi for package: php-mbstring-5.4.45-17.el7.remi.x86_64
--> Finished Dependency Resolution
Error: Package: php-ldap-5.4.45-17.el7.remi.x86_64 (remi)
           Requires: php-common(x86-64) = 5.4.45-17.el7.remi
           Installed: php-common-7.3.5-1.el7.remi.x86_64 (@remi-php73)
               php-common(x86-64) = 7.3.5-1.el7.remi
           Available: php-common-5.4.16-46.el7.x86_64 (base)
               php-common(x86-64) = 5.4.16-46.el7
           Available: php-common-5.4.45-16.el7.remi.x86_64 (remi)
               php-common(x86-64) = 5.4.45-16.el7.remi
           Available: php-common-5.4.45-17.el7.remi.x86_64 (remi)
               php-common(x86-64) = 5.4.45-17.el7.remi
Error: Package: php-mbstring-5.4.45-17.el7.remi.x86_64 (remi)
           Requires: php-common(x86-64) = 5.4.45-17.el7.remi
           Installed: php-common-7.3.5-1.el7.remi.x86_64 (@remi-php73)
               php-common(x86-64) = 7.3.5-1.el7.remi
           Available: php-common-5.4.16-46.el7.x86_64 (base)
               php-common(x86-64) = 5.4.16-46.el7
           Available: php-common-5.4.45-16.el7.remi.x86_64 (remi)
               php-common(x86-64) = 5.4.45-16.el7.remi
           Available: php-common-5.4.45-17.el7.remi.x86_64 (remi)
               php-common(x86-64) = 5.4.45-17.el7.remi
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest

```
Does it work with php 7.x or do I really need a php 5.x?

Thanks for your support.
Michiel.

**Comment by coudot** [comment]
It seems your package manager is configured to use PHP 5.4, you need at least PHP 5.6. Of course this works with PHP 7.

**Comment by mickeybyte** [comment]
Any idea how I can set the package manager to use php 7?

Thanks!

**Comment by coudot** [comment]
Seems you are using remi repo. SSP does not need this repo, you should use the PHP version provided by CentOS 7.

**Comment by mickeybyte** [comment]
I need the remi repo for PHP7. CentOS 7 ships with PHP5. 
I did found the solution! Apparently I had php 7.3.5 base modules, but some the required modules for self-service password were on 7.3.6. I enabled the php73 repo from remi (yum-config-manager --enable remi-php73) and ran yum update, which updated all php modules to 7.3.6. Afterwards, I could install the self-service password without any problems.

Thanks for the quick support!
Michiel.
